src/Repository/CommissionableAssetRepository.php

```diff
@@ -85,6 +85,10 @@ public function findByFilters(array $formData, bool $onlyIds = false): array
             $qb = $this->addAvailabilityBetween($qb, $formData['availableFrom'], $formData['availableTo'], CommissionableAssetAvailability::class, 'asset');
         }
 
+        if (($formData['withoutAvailabilities'] ?? false) === false && empty($formData['availableFrom']) && empty($formData['availableTo'])) {
+            $qb = $this->addAvailabilityBetween($qb, $formData['from'], $formData['to'], CommissionableAssetAvailability::class, 'asset');
```

Comment:
Caution: you cannot use this method here: what we want is `Do not display users without availability in the planning`, not `Display only ressources available ON EVERY slot between from and to`, you must create a new function in order to get assets and users `with AT LEAST one "not red" slot  between from and to`